internal/api/graphql/graph/baseResolver/common.go

```diff
@@ -38,6 +39,7 @@ var (
 	ServiceFilterType             string = "type"
 	ServiceFilterUserName         string = "userName"
 	ServiceFilterSupportGroupCcrn string = "supportGroupCcrn"
+	ServiceFilterUserNameWithId   string = "userNameWithId"
```

Comment:
This needs to be renamed to `uniqueUserId` because that's how the filter is called in `ServiceFilter`